src/naksu/mebroutines/mebroutines_test.go

```diff
@@ -0,0 +1,33 @@
+package mebroutines_test
+
+import "testing"
+import "naksu/mebroutines"
+
+func TestIfIntlCharsInPath (t *testing.T) {
+  tables := []struct {
+    path string
+    result bool
+  }{
+    {"C:\\Users\\john.doe\\ktp-jako", false},
+    {"C:\\Users\\raimo.keski-vääntö\\ktp-jako", true},
+    {"C:\\Users\\john doe\\ktp-jako", false},
+
+    {"/home/someuser/ktp-jako", false},
+    {"/home/öylätti/ktp-jako", true},
+    {"~/ktp-jako", true},
+    {"~root/loremipsum", true},
+
+    {"random whatever string", false},
+    {"random whatever string with öljyrätti", true},
+    {"wtf!", true},
+    {"what?", true},
+    {"/home/ktp-user/*", true},
+  }
+
+  for _, table := range tables {
+    is_intl := mebroutines.IfIntlCharsInPath(table.path)
```

Comment:
naming - Go in general prefers `isIntl` camelCase format instead of `snake_case`. I think linter complains about these.